dub.json

```diff
@@ -64,6 +64,7 @@
                 "source/agora/consensus/Quorum.d",
                 "source/agora/network/Client.d",
                 "source/agora/network/Manager.d",
+                "source/agora/api/Registry.d",
```

Comment:
Okay, I looked more into this. It fails because the moduleinfo of `agora.flash.Node` is missing.
It's missing because it is referenced from the `api` module, as it imports it on Registry.d:L20.
Will comment there.